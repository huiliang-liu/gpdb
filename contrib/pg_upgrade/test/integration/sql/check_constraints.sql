create table users_with_check_constraints (
    id int, 
    name text check (id>=1 and id<2)
);

insert into users_with_check_constraints values (1, 'Joe');
insert into users_with_check_constraints values (2, 'Jane');
