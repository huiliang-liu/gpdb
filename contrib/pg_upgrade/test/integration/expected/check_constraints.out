create table users_with_check_constraints ( id int, name text check (id>=1 and id<2) );
CREATE

insert into users_with_check_constraints values (1, 'Joe');
INSERT 1
insert into users_with_check_constraints values (2, 'Jane');
ERROR:  new row for relation "users_with_check_constraints" violates check constraint "users_with_check_constraints_id_check"
